modules/gitops-uninstall-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="uninstalling-keycloak_{context}"]
```

Comment:
```suggestion
[id="gitops-uninstall-keycloak_{context}"]
```
The `id` value needs to be the same as the filename per the OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata